content/docs/user-guide/managing-external-data.md

```diff
@@ -13,18 +13,14 @@ track data outside of the <abbr>project</abbr>.
 ## How external outputs work
 
 DVC can track existing files or directories on an external location with
-`dvc add` (`out` field). It can also create external files or directories as
-outputs for `dvc.yaml` files (only `outs` field, not metrics or plots).
+`dvc add`. It can also define external outputs for `dvc.yaml` stages to create.
 
 External outputs are considered part of the (extended) DVC project: DVC will
-track changes in them, and reflect this in `dvc status` reports, for example.
+detect when they change, reporting this in `dvc status` for example.
```

Comment:
it's not only about detecting changes, it's about versioning them (that's where we use "track")

Reply:
Good point. But "track when they change" doesn't sound right... I went for "track them for [versioning], thus detecting when they change".